dev/com.ibm.ws.threading/src/com/ibm/ws/threading/ThreadQuiesce.java

```diff
@@ -17,11 +17,12 @@
  */
 public interface ThreadQuiesce {
 
+    @Deprecated
```

Comment:
Should this be deprecated?   Or just use the default 30 sec?

Reply:
> Should this be deprecated? Or just use the default 30 sec?

It's only called from the RuntimeUpdateManager.  So future calls (none expected) should pass in the timeout.  Defaulting to 30 means hard-coding the 30 in yet another place.  It's already hard-coded in the Metatype.

Reply:
> Should this be deprecated? Or just use the default 30 sec?

Actually, I guess I hard-coded it anyway, since the API is still there even though deprecated.   Ideally, I could remove it, but then the API police would come get me.